kotlin/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+plugins {
+    kotlin("jvm")
+    id("org.jetbrains.intellij")
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation(kotlin("stdlib"))
+
+    implementation(project(":langwrappers")) // Interfaces that cover language-specific logic
+    implementation(project(":core"))
+}
+
+intellij {
+    rootProject.properties["platformVersion"]?.let { version.set(it.toString()) }
+    plugins.set(listOf("java", "org.jetbrains.kotlin"))
+}
+
+tasks.named("verifyPlugin") { enabled = false }
+tasks.named("runIde") { enabled = false }
+tasks.named("runPluginVerifier") { enabled = false }
+
+tasks {
+    buildSearchableOptions {
+        enabled = false
+    }
+}
+
+kotlin {
+    jvmToolchain(17)
+}
```

Comment:
This `jvmToolchain(17)` is used multiple times. 

I suggest moving the `17` into a variable on the upper `gradle.build` level and propagating it down to the dependent `gradle.build` scripts.